packages/sdk-client-v3/src/middleware/create-user-agent-middleware.ts

```diff
@@ -14,8 +14,8 @@ export default function createUserAgentMiddleware(
   return (next: Next): Next =>
     async (request: MiddlewareRequest): Promise<MiddlewareResponse> => {
       const userAgent = createUserAgent({
-        ...options,
         name: `commercetools-sdk-javascript-v3/${packageJson.version}`,
+        ...options,
```

Comment:
Is there any reason why you made this change?

Reply:
When the user provides a name, it should override the default name. Therefore I made this change.

Reply:
https://github.com/commercetools/commercetools-sdk-typescript/pull/708/commits/0f40d0dd0110fd858a91073190e8a509772c3c05